docs/UserGuide.md

```diff
@@ -119,13 +119,14 @@ You can delete assignments from your assignment list by specifying the assignmen
 
 You can delete **one or more** assignments at a time. Here is an example with steps to follow:
 
-1) To delete assignments with the name "Statistics tutorial" and "Biology lab report" as shown in the figure below, you can simply enter `delete 1 3` into the command box as per their indexes that are labelled in the figure.
+1) To delete assignments with the name "Statistics tutorial" and "Biology lab report" as shown in the figure below, you can simply enter `delete 1 3` into the Command Box as per their indexes that are labelled in the figure.
 
 ![DeleteCommand1](images/DeleteCommandDiagram1.png)
 *Figure X: User input and location of assignment indexes*
 
-2) The two assignments are no longer displayed and are successfully deleted from your assignment list. 
-3) A "Deleted assignment(s)" message that includes the information of your deleted assignments will be displayed in the Command Box.
+2) You have successfully deleted both assignments from the assignment list, and they are no longer displayed.
+
+3) A "Deleted assignment(s)" message that includes the information of your deleted assignments will be displayed in the Message Box.
```

Comment:
of your deleted assignment(s) ?

Reply:
its correct! Message displayed is "Deleted Assignment(s): Name of assignment ......."